lib/public/IConfig.php

```diff
@@ -249,4 +249,15 @@ public function deleteAppFromAllUsers($appName);
 	 * @since 8.0.0
 	 */
 	public function getUsersForUserValue($appName, $key, $value);
+
+	/**
+	 * Gets the list of users based on their lastLogin info asc or desc
```

Comment:
```suggestion
	 * Gets the list of users sorted by lastLogin, from most recent to least recent
```
Since we can't choose asc or desc, copied from the other method